tools/webhook-apicoverage/resourcetree/display_helper.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resourcetree
+
+import (
+	"github.com/knative/test-infra/tools/webhook-apicoverage/coveragecalculator"
+	"strconv"
+	"strings"
+)
+
+// GetJSONTypeDisplay is a helper method to display API Coverage details in json-like format.
+func GetJSONTypeDisplay(coverageData []coveragecalculator.TypeCoverage, displayRules DisplayRules) string {
```

Comment:
If it's a helper, shouldn't it be private?

Reply:
Also, I don't see it being used here. What's the plan for a file that's not standard, and looks like debugging? Is it worth documenting it?

Reply:
Its actually going to be used by repos, so they would be calling this method from other packages hence it was marked as public. Thinking more about it, this should go into its own package. I have moved this file and DisplayRules into its own package and added a README file to document it.